tests/baselines/reference/api/typescript.d.ts

```diff
@@ -4473,6 +4496,7 @@ declare namespace ts {
     }
     type FileWatcherCallback = (fileName: string, eventKind: FileWatcherEventKind, modifiedTime?: Date) => void;
     type DirectoryWatcherCallback = (fileName: string) => void;
+    type BufferEncoding = "ascii" | "utf8" | "utf-8" | "utf16le" | "ucs2" | "ucs-2" | "base64" | "latin1" | "binary" | "hex";
```

Comment:
I wonder if we could even limit this to future-proof ourselves. But maybe we shouldn't break our heads over it.